lib/client.js

```diff
@@ -311,65 +381,75 @@ const warmupLimit = 32;
  * the routing.
+ * [TODO:]
  * @property {Buffer|Array} [routingKey] Partition key(s) to determine which coordinator should be used for the query.
+ * [TODO:]
  * @property {Array} [routingNames] Array of the parameters names that are part of the partition key to determine the
  * routing. Only valid for non-prepared requests, it's recommended that you use the prepare flag instead.
+ * [TODO:]
  * @property {Number} [serialConsistency] Serial consistency is the consistency level for the serial phase of
  * conditional updates.
  * This option will be ignored for anything else that a conditional update/insert.
+ * [TODO:]
  * @property {Number|Long} [timestamp] The default timestamp for the query in microseconds from the unix epoch
  * (00:00:00, January 1st, 1970).
  * <p>If provided, this will replace the server side assigned timestamp as default timestamp.</p>
  * <p>Use [generateTimestamp()]{@link module:types~generateTimestamp} utility method to generate a valid timestamp
  * based on a Date and microseconds parts.</p>
+ * [TODO:]
  * @property {Boolean} [traceQuery] Enable query tracing for the execution. Use query tracing to diagnose performance
  * problems related to query executions. Default: false.
  * <p>To retrieve trace, you can call [Metadata.getTrace()]{@link module:metadata~Metadata#getTrace} method.</p>
+ * [TODO: Delete?]
  * @property {Object} [graphOptions] Default options for graph query executions.
  * <p>
  *   These options are meant to provide defaults for all graph query executions. Consider using
  *   [execution profiles]{@link ExecutionProfile} if you plan to reuse different set of options across different
  *   query executions.
  * </p>
+ * [TODO: Delete?]
  * @property {String} [graphOptions.language] The graph language to use in graph queries. Default:
  * <code>'gremlin-groovy'</code>.
+ * [TODO: Delete?]
  * @property {String} [graphOptions.name] The graph name to be used in all graph queries.
  * <p>
  * This property is required but there is no default value for it. This value can be overridden at query level.
  * </p>
+ * [TODO: Delete?]
  * @property {Number} [graphOptions.readConsistency] Overrides the
  * [consistency level]{@link module:types~consistencies}
  * defined in the query options for graph read queries.
+ * [TODO: Delete?]
  * @property {Number} [graphOptions.readTimeout] Overrides the default per-host read timeout (in milliseconds) for all
  * graph queries. Default: <code>0</code>.
  * <p>
  *   Use <code>null</code> to reset the value and use the default on <code>socketOptions.readTimeout</code> .
  * </p>
+ * [TODO: Delete?]
  * @property {String} [graphOptions.source] The graph traversal source name to use in graph queries. Default:
  * <code>'g'</code>.
+ * [TODO: Delete?]
  * @property {Number} [graphOptions.writeConsistency] Overrides the [consistency
  * level]{@link module:types~consistencies} defined in the query options for graph write queries.
  */
 
 /**
```

Comment:
When I generated the docs from in-code docs I think this whole upper section describing the parameters is lost (at least for me).